static/src/js/util/url/__tests__/projectCollections.test.js

```diff
@@ -441,6 +442,6 @@ describe('url#encodeUrlQuery', () => {
       }
     }
 
-    expect(encodeUrlQuery(props)).toEqual('/path/here?p=!collectionId1!collectionId2&pg[1][v]=f&pg[1][m]=harmony&pg[1][ets]=f&pg[2][v]=f')
+    expect(encodeUrlQuery(props)).toEqual('/path/here?p=!collectionId1!collectionId2&pg[1][v]=f&pg[1][m]=harmony&pg[1][ets]=f&pg[1][ess]=f&pg[2][v]=f')
```

Comment:
This test is named `correctly encodes enable temporal subsetting`, you should pull this field out of this test and test it independently